css/navbar.css

```diff
@@ -0,0 +1,11 @@
+nav.hide {
```

Comment:
the `hide` / `show` class is too general . please concretize the class name. `nav-show-block` ,  `nav-hide-block`, etc.